src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+import { User } from '../../types/User';
+
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user: { name, email } }) => (
+  <>
+    <a className="UserInfo" href={`mailto:${email}`}>
+      {name}
+    </a>
+  </>
```

Comment:
unnecessary fragment
```suggestion
  <a className="UserInfo" href={`mailto:${email}`}>
    {name}
  </a>
```